### DISCUSSION TITLE
cli gives permission error

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#3822] [General]
hi everyone

all installed and working fine.. tried out the cli and no matter what command I try I get an error
```/opt/photoprism/bin/photoprism users```
`ERRO[2023-10-14T16:20:46+10:00] invalid originals path, check configuration and permissions `

using 
```
/opt/photoprism/bin/photoprism start
/opt/photoprism/bin/photoprism status
```
these work.. its starts.. gives me the status `operational`

paths are all there, I can access them a ls files.. so not sure why the cli is not working.. nothing is showing the /lib/errors area

**Comment by lastzero** [comment]
You should be able to run `photoprism config` to see all config values for troubleshooting.

**Comment by ghost** [comment]
thanks

I did notice that the `originals-path` is blank in the config output, I do have one set in the .env file
`PHOTOPRISM_ORIGINALS_PATH='/var/lib/photoprism/photos/Originals'`

I didn't see a set option for config items so im guessing a passed in arg on `start`

seems that none of these files exist
```
| options-yaml           | /opt/photoprism/bin/.photoprism/storage/config/options.yml                                  |
| defaults-yaml          | /etc/photoprism/defaults.yml                                                                |
| settings-yaml          | /etc/photoprism/settings.yml (defaults)                                                     |
| settings-yaml          | /opt/photoprism/bin/.photoprism/storage/config/settings.yml    
```

instead of env items should I move items into new files as above?

**Reply by lastzero** [reply]
Well, if the env variables do not get picked up (would need more knowledge about your local environment to guess why this is happening), you can simply put the config in these yaml files as described in our Getting Started docs 👍

**Reply by lastzero** [reply]
https://docs.photoprism.app/getting-started/config-files/

**Comment by ghost** [comment]
I bet this will fix it:)
